src/main/java/ru/timeconqueror/spongemixins/MinecraftURLClassPath.java

```diff
@@ -50,18 +52,38 @@
     /**
      * Get a jar within the minecraft mods directory
      */
+    @SuppressWarnings("All")
     public static File getJarInModPath(final String jarname) {
         try {
-            return java.nio.file.Files.walk(new File(Launch.minecraftHome, "mods/").toPath()).filter( p -> {
-                final String filename = p.toString();
-                return Files.getNameWithoutExtension(filename).contains(jarname) && Files.getFileExtension(filename).equals("jar");
-            }).map(Path::toFile).findFirst().orElse(null);
+            return java.nio.file.Files.walk(MOD_DIRECTORY_PATH)
+                .filter( p -> {
+                    final String filename = p.toString();
+                    return Files.getNameWithoutExtension(filename).contains(jarname) && Files.getFileExtension(filename).equals("jar");
```

Comment:
This will not load liteloader mods (custom extension). The ExampleMod/example-mixins shows how to add it.